e2e-tests/config/common.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * get Device Name from script
+ * @param platform iOS or Android
+ */
+const getDeviceName = (platform) => {
+  const deviceName = process.argv.find((arg) => arg.includes('--deviceName'));
+  const defaultDeviceName = platform === 'iOS' ? 'iPhone 13' : 'Pixel 3a API 33';
+  return deviceName ? deviceName.split('=')[1] : defaultDeviceName;
+};
+
+/**
+ * get Platform Version from script
+ * @param platform iOS or Android
+ */
+const getPlatformVersion = (platform) => {
+  const platformVersion = process.argv.find((arg) => arg.includes('--platformVersion'));
+  const defaultPlatformVersion = platform === 'iOS' ? '15.0' : '13';
+  return platformVersion ? platformVersion.split('=')[1] : defaultPlatformVersion;
+};
```

Comment:
I think the way you have it split up currently is really good. The only suggestion I might make is for these functions to be split down even further.
Perhaps `getDeviceName(platform)` still exists, but instead of doing the logic for `'iOS' ? 'iPhone 13' : 'Pixel 3a API 33';` in here, we create an additional function to say `getAndroidDeviceName` and `getiOSDeviceName`, and then `getDeviceName` calls the appropriate one. This way we can modularize the iPhone/Android versions

Just a suggestion though! I am not too familiar with Appium or e2e testing strategies, and looking at what you've written so far looks great!!! 😊

Reply:
The flexibility provided by `deviceName` and `platformVersion` seems adequate to me.
Clever handling of this, great job!

Reply:
The one suggestion that I might have is to put these versions into a separate setup file, similar to all the other versions that we need in our build https://github.com/e-mission/e-mission-phone/blob/master/setup/export_shared_dep_versions.sh

so:
1. we can upgrade without changing the code
2. people can find all the versions in one place instead of wading through tests to find them.